pkgs/development/haskell-modules/configuration-common.nix

```
@@ -1027,10 +1027,18 @@ self: super: {
   # https://github.com/mgajda/json-autotype/issues/25
   json-autotype = dontCheck super.json-autotype;
 
-  # Requires pg_ctl command during tests
+  # Aeson dependency version bound not satifised but it builds fine without it
+  beam-core = doJailbreak super.beam-core;
+
+  # Aeson dependency version bound not satifised but it builds fine without it
+  beam-migrate = doJailbreak super.beam-migrate;
+
+  # dontcheck: Even though pg_ctl command is provided during tests it doesn't work.
   beam-postgres = overrideCabal super.beam-postgres (drv: {
+    jailbreak = true;
+    doCheck = false;
     testToolDepends = (drv.testToolDepends or []) ++ [pkgs.postgresql];
-    });
+    })));
```

These parens are now too much, I forgot to include that line in my suggestion.

```suggestion
    });
```